portal/components/config.js

```diff
@@ -41,7 +58,7 @@ const resourceConfig = {
         en: "Frequently Asked Questions",
         hi: "जानकारी",
       },
-      target: "/#",
+      target: "https://drive.google.com/file/d/1uKVDD2_c94I3qU85VEJrlKx4I8VDd0lp/view",
```

Comment:
@sagar-techjoomla - Please use an environment variable instead of adding the URL in the code